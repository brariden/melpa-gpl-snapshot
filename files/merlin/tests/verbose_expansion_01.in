#!/bin/bash
. functions.sh || exit 1

reset sexp_01.ml
tell <<END
type t = int * string
let x : t = 5, "lol"
END
cmd type enclosing '{"expr":"x","offset":0}' `pos 2 5`
cmd type enclosing '{"expr":"x","offset":0}' `pos 2 5`
