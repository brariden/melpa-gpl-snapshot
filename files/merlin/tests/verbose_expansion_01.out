["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":3,"col":0},"marker":false}]
["return",{"cursor":{"line":3,"col":0},"marker":false}]
["return",[{"start":{"line":2,"col":4},"end":{"line":2,"col":5},"type":"t","tail":"no"}]]
["return",[{"start":{"line":2,"col":4},"end":{"line":2,"col":5},"type":"int * string","tail":"no"}]]
